### ISSUE TITLE
0/1 kerning varies unexpectedly in certain places on iPhone

### ISSUE THREAD

**Issue by floatingatoll** [designker/ishtar-commander-bugs#138]
In some places in the app, "0/1" or "1/30" isn't kerned the same as other places in the app. Specifically, in the screenshot below, you can see that the "/" character hasn't kerned into the blank space to the right of the 1, when it would have if it was on the main screen as an item count. so this ends up causing narrow numbers to look weird when combined with "x/y" presentation, but only sometimes. Maybe it's just an artifact of the bounty font?

Screenshots are from build 603. I've noticed this issue for a long time now but never got around to filing a bug until now (sorry). It's existed in plain English for as long as I've been using Ishtar, I think.

![photo_ - 2 5](https://cloud.githubusercontent.com/assets/92684/17566903/0bc448f4-5ef2-11e6-9e4e-a0f193a5d530.png)
![photo_ - 1 25](https://cloud.githubusercontent.com/assets/92684/17566904/0be54f72-5ef2-11e6-804a-efe2aa1ea8f8.png)

**Comment by designker** [comment]
The framework I use has zero kerning support. Closed for now.
